src/fp6.rs

```diff
@@ -273,6 +312,49 @@ impl Fp6 {
         }
     }
 
+    #[cfg(all(target_os = "zkvm", target_arch = "riscv32"))]
+    #[inline]
+    fn mul_interleaved(&self, b: &Self) -> Self {
+        let a = self;
+        let b10_p_b11 = b.c1.c0.add_zkvm(&(b.c1.c1));
```

Comment:
It would simplify this part of the code if this could be an add instead of an add_zkvm. But like I said in my earlier comment, I may be missing something about why that design is needed.